l10n_it_corrispettivi/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2016 Lorenzo Battistini - Agile Business Group
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    'name': 'Italian Localization - Corrispettivi',
+    'version': '11.0.1.0.0',
```

Comment:
please add https://docs.google.com/document/d/1MmGxv0yTw0pvB-PYAcUhbCcku9ZG-Y2DSGXwZDNYyIw according to https://docs.google.com/document/d/1MmGxv0yTw0pvB-PYAcUhbCcku9ZG-Y2DSGXwZDNYyIw

Reply:
Done in https://github.com/OCA/l10n-italy/pull/585/commits/befb42f6223940b74cdd0e8490e57dde06633e0c

Reply:
:+1:  I also checked https://github.com/SimoRubi/l10n-italy/blob/11.0-mig-l10n_it_corrispettivi/l10n_it_corrispettivi/README.rst and it doesn't need to be updated, probably 'cause Beta is the default for `development_status`.